.gitignore

```diff
@@ -1,5 +1,6 @@
 ## common
 __pycache__
+.gitignore
```

Comment:
Please don't check .gitignore into .gitignore. We need to know when this file changes.